docker-compose-mysql.yml

```diff
@@ -56,10 +56,7 @@ services:
       KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
 
   mysql:
-    build:
-      context: ./mysql
-      args:
-        EVENTUATE_COMMON_VERSION: "$EVENTUATE_COMMON_VERSION"
+    image: eventuateio/eventuate-tram-sagas-mysql:0.13.0.RELEASE
```

Comment:
`0.13.0.RELEASE` should be `$eventuateTramSagasImageVersion`

Reply:
Fixed